src/controllers/cartsController.js

```diff
@@ -27,4 +28,21 @@ const getCartsController = async (req, res) => {
   }
 };
 
-module.exports = { addItemToCartsController, getCartsController };
+const deleteProductController = async (req, res) => {
```

Comment:
```suggestion
const deleteProduct = async (req, res) => {
```
controller 모듈에 잇으니 따로 적을 필요는 없어보입니다!